addons/escoria-core/game/core-scripts/esc_room.gd

```diff
@@ -78,7 +87,24 @@ func _ready():
 	
 	if global_id.empty():
 		global_id = name
-
+		
+	if run_events and esc_script:
```

Comment:
Why is this called here and in change_room as well?

Reply:
The `run_events` variable determines whether the events are run here or not. The variable is true if the game scene was not loaded. If it was already loaded, that means the scene is loaded from change_scene command.

The idea was to differentiate the events execution when the room is started from change_scene or where the room is directly run.